src/properties/common.props

```diff
@@ -37,7 +36,7 @@
   </PropertyGroup>
 
   <!-- Enable delay signing with Key.snk -->
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(DelaySign)' == 'true'">
```

Comment:
>DelaySign [](start = 31, length = 9)

Update your release job to set this to true

Reply:
Already updated.